Setup.bat

```diff
@@ -57,7 +57,10 @@ call :MarkStartOfBlock "Setup variables"
     set SCHEMA_COPY_DIR=%~dp0..\..\..\spatial\schema\unreal\gdk
     set SCHEMA_STD_COPY_DIR=%~dp0..\..\..\spatial\build\dependencies\schema\standard_library
     set SPATIAL_DIR=%~dp0..\..\..\spatial
-
+    set DOMAIN_ENVIRONMENT_VAR=
+    for %%A in (%*) do (
+        if "%%A"=="--China" set DOMAIN_ENVIRONMENT_VAR=--domain spatialoschina.com --environment cn-production

```

Comment:
```suggestion
        if /i "%%A"=="--China" set DOMAIN_ENVIRONMENT_VAR=--domain spatialoschina.com --environment cn-production
```

Reply:
I did have this originally but it didn't have parity with the shell script - what do you think? :D

Reply:
Hmm in that case I would instead go for lowercase `--china` only.